data/json/furniture.json

```diff
@@ -2896,6 +2915,7 @@
     "type": "furniture",
     "id": "f_egg_sackbw",
     "name": "spider egg sack",
+    "description": "Kind of icky.  Something IS moving in there.",
```

Comment:
If you feel compelled to be arachnophobia-inducing, you can try to describe the differences between the three egg sacs.

f_egg_sackbw is for black widows, f_egg_sackcs is for cellar spiders and f_egg_sackws is for wolf spiders (which generally are on or VERY close to the female wolf spider [they hatch onto her abdomen]).

[Black Widow w/ egg sac](https://ucanr.edu/blogs/bugsquad//blogfiles/24303_original.jpg).

[Cellar Spider w/ egg sac](https://spiderid.com/picture/1742/).

[Wolf Spider w/ egg sac](https://bugguide.net/node/view/143934).

Reply:
Done